balaca86-chennai.md

```diff
@@ -0,0 +1,11 @@
+## Chennai, TN
+
+
+## Great places to eat
+- Hotel Saravana Bhavan
+- Residency Towers
+- Bikes and Barrels
+## Things to do
+- Marina Beach
```

Comment:
@leefp It is the natural urban beach which is 4 mi long, which makes it India's longest beach!!